src/content/en/2021/caching.md

```diff
@@ -17,4 +20,456 @@ featured_stat_3: TODO
+
+Finally, HTML has the largest 1st party group served with less than a week with 42.7% and third party's largest group is between 1-7 weeks with 43.1%. 
+
+Considerations after reviewing this data: 
+- The freshest content for 1st party is HTML while for 3rd party it is CSS. 
+- The most stale content for both 1st and 3rd party is Images.
+
+This data shows us that 1st parties have prioritized refreshing HTML content, which usually holds the link to JS and CSS files, while 3rd party providers that are mostly CSS and script-driven, like browser extensions, have prioritized keeping their CSS up-to-date. When we consider the origins behind 1st parties vs. 3rd parties, it follows that the _way_ content is delivered may be more important to 3rd parties than the actual content, thus making their presentation and optimization of it, all the more important. 
+
+Mobile resources with a cache TTL that was considered too short compared to its content age have seen an improvement since 2020. This data is exciting because it hints at the community's growing understanding of appropriate caching. While a cache TTL that is too long is unhelpful, it is also less useful if it is too short. The correlation between cache TTL and content age is slowly closing the gap, moving from 60.2% in 2020 to 54% in 2021. The more we can pay attention to content's age, the more accurately we can set cache limits.
+
+{{ figure_markup(
+  caption="TODO",
+  content="54%",
+  classes="big-number",
+  sheets_gid="768623684",
+  sql_file="todo.sql"
+) }}
+
+Developers are getting better at setting the cache duration more accurately to the content age, resulting in more responsible, and therefore more effective, caching.
+
+<figure>
+  <table>
+    <tr>
+     <th>Client</th>
+     <th>1st party</th>
+     <th>3rd party</th>
+     <th>Overall</th>
+    </tr>
+    <tr>
+     <td>desktop</td>
+     <td class="numeric">59.3%</td>
+     <td class="numeric">46.2%</td>
+     <td class="numeric">54.3%</td>
+    </tr>
+    <tr>
+     <td>mobile</td>
+     <td class="numeric">60.1%</td>
+     <td class="numeric">44.7%</td>
+     <td class="numeric">54.3%</td>
+    </tr>
+  </table>
+  <figcaption>TODO: Caption the table above</figcaption>
+</figure>
+
+When we split the data between first party and third party providers, the largest improvements come from third party where we have a 13.2% improvement. It is highly encouraging to see how companies around the world are building products for developers that are highly optimized on caching. However, the challenge remains for how first parties can effectively improve over the coming years.
+
+The community's increased attention on improving performance has encouraged and even incentivized third parties to optimize their caching strategies.
+
+## Identifying caching opportunities
+
+Based on the **Lighthouse caching TTL* score, we have seen an improvement in pages ranked with a perfect 100 score increase from 3.3% in 2020 to 4.4% in 2021.
```

Comment:
Edits: 

Based on the **Lighthouse caching TTL* score, we have seen an improvement in pages ranked with a perfect score of 100 increase from 3.3% in 2020 to 4.4% in 2021.

The score reflects whether the pages can benefit from additional caching policy improvements. Even though we are excited to see 31% of pages scoring above the 50 percentile score, a large potential exists for the 52% of pages that are ranking below the 25 percentile.

{{ figure_markup(
  image="lighthouse-caching-ttl-scores.png",
  caption="Distribution of Lighthouse caching TTL scores.",
  description="TODO",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=1825783189&format=interactive",
  sheets_gid="2098771743",
  sql_file="cache_ttl_lighthouse_score.sql"
) }}

This makes us consider that even though web pages have some level of caching, the way the policies are used is outdated and not optimized to the latest state of their products.

Based on **Lighthouse wasted bytes** from 2020 to 2021, there was a 3.28% improvement in wasted bytes across all audited pages on repeated views. This lowers the percentage of pages that waste 1 MB from 42.8% to 39.5%, showing a considerable trend from the community towards building products that are less costly for international users with paid internet data plans.

{{ figure_markup(
  image="lighthouse-caching-byte-savings.png",
  caption="Distribution of potential byte savings from caching.",
  description="TODO",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=1045307868&format=interactive",
  sheets_gid="469776025",
  sql_file="cache_wastedbytes_lighthouse.sql"
) }}

Something we're looking forward to seeing in the coming years is an increase in the percentage of pages that have 0 wasted bytes, which at the moment is only at 1.34%.